### ISSUE TITLE
lint errors by custom ruleset are reported with wrong Reference

### ISSUE THREAD

**Issue by toschder** [wework/speccy#228]
When using custom ruleset, the lint errors are always reported with wrong links to "More information":

```
[31mSpecification contains lint errors: 7[0m

[33m#/info [36m R: API_REPO_VERS_SEMANTIC [37m D: MUST: Use Semantic Versioning (info.version)
[0mMUST: Use Semantic Versioning (info.version)

More information: https://speccy.io/rules/1-rulesets#API_REPO_VERS_SEMANTIC

```
## Detailed description
This is my rule:
`        {
            "name": "API_REPO_VERS_SEMANTIC",
            "object": "info",
            "enabled": true,
            "description": "MUST: Use Semantic Versioning (info.version)",
            "pattern": { "property": "version",
                         "value": "^[0-9]+[.][0-9]+([.][0-9]+)?$"
                       }
        },
`
And it was invoked with
`speccy lint /path/to/swagger2openapi.file -v --rules speccy/mycompany-rules.json`

I would propose to name the ruleset like given with the --rules option. Another way could be to be able to define a URL in the custom-rules.json File(s) being given.

## Context

I think everyone could benefit from that. The first thing you usually will do when hitting a validation error is to look into the rule(s) - and the available description,examples and so on.

## Your environment

Ubuntu, speccy 0.8.x (latest)

**Comment by MikeRalphson** [comment]
`oas-kit` will have this functionality in the next release. In addition to the above suggested implementation, `rules.url` may be overridden at the individual rule level. It is expeced that the user of the linter will gather a unique set of `rule.url`+`rule.name` violations to output.

**Comment by MikeRalphson** [comment]
@toschder please may I add your rule to the `oas-kit` wiki?

**Comment by djtarazona** [comment]
This has been fixed in [v0.10.0](https://github.com/wework/speccy/releases/tag/v0.10.0)
